src/Physics/physicsHelper.ts

```diff
@@ -19,29 +19,37 @@ export class PhysicsHelper {
 
     private _scene: Scene;
     private _physicsEngine: Nullable<IPhysicsEngine>;
+    public options: PhysicsHelperOptions;
 
     /**
      * Initializes the Physics helper
      * @param scene Babylon.js scene
      */
-    constructor(scene: Scene) {
+    constructor(scene: Scene, options: PhysicsHelperOptions) {
         this._scene = scene;
         this._physicsEngine = this._scene.getPhysicsEngine();
 
         if (!this._physicsEngine) {
             Logger.Warn('Physics engine not enabled. Please enable the physics before you can use the methods.');
         }
+
+        this.options = {...(new PhysicsHelperOptions()), ...options};
+
+        if (this.options.useNativePhysicsRaycastingIfAvailable) {
+          if (this._physicsEngine.getPhysicsPluginName() === 'OimoJSPlugin') {
+            this.options.useNativePhysicsRaycastingIfAvailable = false;
+            Logger.Warn('Native raycasting not available on the OimoJSPlugin.');
+          }
+        }
     }
 
     /**
      * Applies a radial explosion impulse
      * @param origin the origin of the explosion
-     * @param radius the explosion radius
-     * @param strength the explosion strength
-     * @param falloff possible options: Constant & Linear. Defaults to Constant
+     * @param eventOptions the options of radial explosion
      * @returns A physics radial explosion event, or null
      */
-    public applyRadialExplosionImpulse(origin: Vector3, radius: number, strength: number, falloff: PhysicsRadialImpulseFalloff = PhysicsRadialImpulseFalloff.Constant): Nullable<PhysicsRadialExplosionEvent> {
+    public applyRadialExplosionImpulse(origin: Vector3, eventOptions: PhysicsRadialExplosionEventOptions): Nullable<PhysicsRadialExplosionEvent> {

```

Comment:
Yes, I know. That's what I mentioned above. The thing is, for now, extending it would work (at least for `applyRadialExplosionImpulse` & co., as it only has 4 attributes (MAYBE more in future), but the votex even now has 7 attributes, and that's probably too much, especially as you may want to edit only some.
Because of that, it makes more sense to ever event just have an options variable added, instead of the vortex event having options, but other event having each parameter separately.